examples/erc20/Compound/supply.en.js

```diff
@@ -16,8 +16,8 @@ class Token {
     render() {
         const decimals18 = 1e+18;
         const decimals8 = 1e+8; //cToken decimals
-        let cTokenBalance = (this.props.cTokenBalance / decimals8).toFixed(2);
-        let tokenBalance = (this.props.tokenBalance / decimals18).toFixed(2);
+        let cTokenBalance = Math.floor((this.props.cTokenBalance / decimals8) * 100) / 100;
```

Comment:
Is it because we want it to be a number rather than a string? If so we can do: 

```
let cTokenBalance = Number((this.props.cTokenBalance / decimals8).toFixed(2));
```

Getting rid of the magic numbers and calculation.

Reply:
No, it's to round down the number